pkg/controllers/vdb/restart_reconciler.go

```diff
@@ -436,7 +436,9 @@ func (r *RestartReconciler) restartCluster(ctx context.Context, downPods []*PodF
 		)
 	}
 	for i := range downPods {
-		opts = append(opts, startdb.WithHost(downPods[i].podIP))
+		if !downPods[i].isTransient {
```

Comment:
The `findRestartablePods` function has a filter for transient pods (the second parameter). This filter is included for cluster restarts, which makes sense because re-ip is necessary to get the nodes in the admintools.conf with the correct IPs. However, this is not necessary for vclusterops, as there is no admintools.conf. Therefore, I recommend modifying the call to `findRestartablePods` for vclusterops to exclude transient pods.